nuxeo-services/nuxeo-platform-dublincore/src/main/java/org/nuxeo/ecm/platform/dublincore/listener/DublinCoreListener.java

```diff
@@ -79,9 +81,9 @@ public void handleEvent(Event event) {
         }
         String eventId = event.getName();
 
-        if (!eventId.equals(DOCUMENT_CREATED) && !eventId.equals(BEFORE_DOC_UPDATE)
-                && !eventId.equals(TRANSITION_EVENT) && !eventId.equals(DOCUMENT_PUBLISHED)
-                && !eventId.equals(DOCUMENT_CREATED_BY_COPY)) {
+        if (!eventId.equals(DOCUMENT_CREATED) && !eventId.equals(BEFORE_DOC_UPDATE) && !eventId.equals(TRANSITION_EVENT)
+                && !eventId.equals(DOCUMENT_TRASHED) && !eventId.equals(DOCUMENT_UNTRASHED)
+                && !eventId.equals(DOCUMENT_PUBLISHED) && !eventId.equals(DOCUMENT_CREATED_BY_COPY)) {
```

Comment:
Shouldn't `nxdublincore-service.xml` be changed also to add the trashed events as well? Is this correctly unit tested?